src/org/jitsi/service/neomedia/RawPacket.java

```diff
@@ -1519,6 +1519,39 @@ public void setOriginalSequenceNumber(int sequenceNumber)
     }
 
     /**
+     * Sets the padding length for this RTP packet.
+     *
+     * @param len the padding length.
+     * @return the number of bytes that were written, or -1 in case of an error.
+     */
+    public int setPaddingSize(int len)
```

Comment:
that's true, but it still wouldn't enable this sort of calling:
```
byte[] rtp = new byte[12]; int off = 0;
off += RawPacket.setVersion(rtp, off, rtp.length);
off += RawPacket.setPayloadType(rtp, off, rtp.length, (byte) 100);
...
```
it'd have to be:
```
byte[] rtp = new byte[12]; int off = 0; int res = 0;
res = RawPacket.setVersion(rtp, off, rtp.length);
if (res > -1) {
  off += res;
} else {
  // handle error
}
...
```
to get that, we could have it return the bytes written if successful or throw if there's an error (that would allow the code in your example).  or it could return bool and modify offset by the amount of bytes written?  that would look like:

```
byte[] rtp = new byte[12]; int off = 0;
if (!RawPacket.setVersion(rtp, off, rtp.length)) {
  // handle error
}
if (!RawPacket.setPayloadType(rtp, off, rtp.length, (byte) 100)) {
  // handle error
}
...

```